docs/manifest_by_file.md

```diff
@@ -267,6 +267,7 @@ File Path | Manifest
 /var/opt/mssql/log/errorlog\* | sql-iaas 
 /var/opt/mssql/setup\* | sql-iaas 
 /var/opt/omi/log/\*.log | monitor-mgmt 
+/var/run/azure-cns/\* | aks 

```

Comment:
anything in /var/run will not get collected by diskinspect-service since /var/run is mounted to a tmpfs that is not actually on block storage

Reply:
Thank you. I'm gonna close the PR.